docs/bypass_adblock_detection.md

```diff
@@ -0,0 +1,16 @@
+# Bypass ad blockers detection
+
+Some ad blockers might detect and block this library because of the known CDN URL or the script name.
+To bypass the specific detection, one needs to check the applied rules.
+Here are some generic options to circumvent this behavior.
+
+## Open-source alternatives
+- Use the [NPM installation option](https://github.com/fingerprintjs/fingerprintjs/blob/master/docs/api.md#webpackrollupnpmyarn) for the open source FingerprintJS. Additionally, make sure the script is not served from the `*/fingerprint.js` path.
+- The less preferred way is to self-host the script on your own domain. It will make your script quickly outdated and it will stop working with new browsers as they get released in the future.
+You can find scripts on the following URLs (replace the version with the current version e.g. with `v3.3.3`).
+  - ES Module download URL: `https://openfpcdn.io/fingerprintjs/v3.3.3/esm.min.js`.
+  - Universal Module Definition download URL: `https://openfpcdn.io/fingerprintjs/v3.3.3/umd.min.js`.
```

Comment:
Version-unspecific URLs should be used:

```suggestion
  - ES Module download URL: `https://openfpcdn.io/fingerprintjs/v3/esm.min.js`.
  - Universal Module Definition download URL: `https://openfpcdn.io/fingerprintjs/v3/umd.min.js`.
```

Reply:
"UMD" is known better than "Universal Module Definition". Maybe it will be unclear for unexperienced developers which version to download.

Reply:
Fixed in https://github.com/fingerprintjs/fingerprintjs/pull/782/files#diff-ca75138eebfa7215a976ae8804b36955e154c883deafffc140fd52580b8631dc

Reply:
Looks good now. Could you please also add a link to this article to the API guide or to the readme?

Reply:
- After a tiny internal survey, I decided to change `bypass` to `evade` wording. I renamed all occurrences.
- I added ad blockers evasion to the feature matrix. The most natural place to me was in the `Additional Features` section right before the `Geolocation`.
- Re: adding a link. I didn't want to add a new unnecessary section to the readme. Also, adding to the API guide is too deep IMO. I decided to add a link to the feature matrix. Once somebody looks for `ad blockers` they immediately see the link.